packages/itwin/grouping-mapping-widget/src/components/Groups/GroupListItem.tsx

```diff
@@ -16,12 +16,12 @@ import { Anchor, ListItem } from "@itwin/itwinui-react";
 import "./GroupListItem.scss";
 
 export interface GroupListItemProps extends Omit<GroupsProps, "onClickAddGroup"> {
-  group: Group;
+  group: Group | GroupMinimal;
```

Comment:
`Group` extends `GroupMinimal`, so `GroupMinimal` should be enough here.

Reply:
Change the rest if possible.

Reply:
Refactored the rest to just `GroupMinimal` as well.